Source/mesquite/lib/StringUtil.java

```diff
@@ -614,8 +592,7 @@ else if (line.equals(""))
 	public static String reverse(String s) {
 		if (s==null)
 			return null;
-		StringBuffer sb = new StringBuffer(s);
-		return sb.reverse().toString();
+		return StringUtils.reverse(s);
```

Comment:
Daisie: isn't this recursive?  It looks to me as if you are asking the function to call itself here.